src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/Nav/BitNavDemo.razor

```diff
@@ -23,7 +23,7 @@
                    box-sizing: border-box;
                    border: 1px solid #eee;
                    overflow-y: auto;"
-                    Class="bit-nav-wrapped-link"
+                    Class="wrapped-link"
```

Comment:
hmm, I don't think this change is correct since it's a new CSS class defined in the demo project and not one of the component's classes.